Tests/MessagingInApp/Core/IntegrationTest.swift

```diff
@@ -73,6 +158,18 @@ extension IntegrationTest {
 
 @MainActor
 extension IntegrationTest {
+    func onCustomActionButtonPressed(onInlineView inlineView: InAppMessageView) {
+        // Triggering the custom action button on inline message from the web engine
+        // mocks the user tap on custom action button
+        getWebEngineForInlineView(inlineView)?.delegate?.tap(name: "", action: "Test", system: false)
+    }
+
+    func onDeepLinkActionButtonPressed(onInlineView inlineView: InAppMessageView, deeplink: String) {
+        // Triggering the custom action button on inline message from the web engine
+        // mocks the user tap on custom action button
+        getWebEngineForInlineView(inlineView)?.delegate?.tap(name: "", action: deeplink, system: true)
+    }
```

Comment:
This is not new code, but code moved from `Tests/MessagingInApp/Views/InAppMessageViewTest.swift` into this function to try and keep all button click functions in 1 place for re-usability.